### PR TITLE
Roslyn 3.4

### PR DESCRIPTION
**Pull request by rchande**

### CODE REVIEW

**Comment by filipw** [comment]
can we please merge https://github.com/OmniSharp/omnisharp-roslyn/pull/1596 first? i'd like to have a separate commit on master on 3.3.1 just in case we want to go back at any point

File: tests/OmniSharp.MSBuild.Tests/ProjectWithAnalyzersTests.cs [hunk 1]
```diff
@@ -32,6 +32,8 @@ public async Task WhenProjectIsRestoredThenReanalyzeProject()
             {
                 await host.RestoreProject(testProject);
 
+                await Task.Delay(2000);
```
[the remark this answers is not in the document]
Comment (rchande) [reply tests/OmniSharp.MSBuild.Tests/ProjectWithAnalyzersTests.cs]: Same as line 137

**Comment by rchande** [comment]
@filipw Sorry, didn't see that PR. Mind we we just supersede it with this one (given that this is a later version)?

File: tests/OmniSharp.MSBuild.Tests/ProjectWithAnalyzersTests.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (rchande) [reply tests/OmniSharp.MSBuild.Tests/ProjectWithAnalyzersTests.cs]: (It's a race condition that causes a test failure)

**Comment by filipw** [comment]
can we not merge that other PR first and just publish a release? then we'd have an OmniSharp release based on 3.3.1 which matches the latest VS preview

then we merge this PR to get one onto 3.4, I think that would be most desirable course of action

File: tests/OmniSharp.MSBuild.Tests/ProjectWithAnalyzersTests.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (filipw) [reply tests/OmniSharp.MSBuild.Tests/ProjectWithAnalyzersTests.cs]: ok i see now, makes sense it's actually used all over the place there 👍

**Comment by rchande** [comment]
From the C# extension side we're likely going to try to align with .Net Core 3.1 preview 1, which this build covers, in the next 2 weeks. I doubt we'll ship another release sooner than that. Does that make sense?

**Comment by filipw** [comment]
shouldn't we wait for a later 3.4 build then?

**Comment by filipw** [comment]
actually we can update again at any point - thanks

**Comment by rchande** [comment]
@filipw Thanks for accommodating us :)

**Pull request merged**